docker-compose.yml

```diff
@@ -0,0 +1,40 @@
+version: '3.8'
```

Comment:
```suggestion
```
Remove version field as in latest update docker has made it obsolete, removing the line fixes the issue, link for reference: https://forums.docker.com/t/docker-compose-yml-version-is-obsolete/141313